lua/lls.lua

```diff
@@ -1,41 +1,49 @@
 return function (net_conf, numa_table)
 
-	-- Init the LLS configuration structure.
+	--
+	-- Configure the variables below for the LLS block.
+	--
+
+	-- These parameters should likely be initially changed.
+	local log_level = staticlib.c.RTE_LOG_DEBUG
+
+	-- XXX #155 These parameters should only be changed for performance reasons.
+	local mailbox_max_entries_exp = 7
+	local mailbox_mem_cache_size = 0
+	local mailbox_burst_size = 32
+	local log_ratelimit_interval_ms = 5000
+	local log_ratelimit_burst = 10
+	local front_max_pkt_burst = 32
+	local back_max_pkt_burst = 32
+	local mailbox_max_pkt_burst = 32
```

Comment:
Shouldn't `mailbox_max_pkt_burst` be grouped with the other mailbox parameters above? If so, update the assignment of `lls_conf.mailbox_max_pkt_burst` below as well as [the documentation](https://github.com/AltraMayor/gatekeeper/wiki/Functional-Block:-LLS).